entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
> Some topics aimed at relatively advanced users don't mention the usings, assuming the reader knows or can easily find them.

Turns out without VS making it easy to find the right using, may xplat customers struggle with them. That's why I've been using the unobtrusive code comment.